tests/Unit/ControlSystem/Tags/Test_MeasurementTimescales.cpp

```diff
@@ -18,6 +18,8 @@
 #include "ControlSystem/Tags/MeasurementTimescales.hpp"
```

Comment:
Sorry, I'm not familiar with these tests. Where would the test fail now if the the changes in this PR weren't made? It looks like most things are `static_assert` but the code changes make a runtime change to data. Is there a way to verify the runtime change occurred?

Reply:
I edited some tests so it should catch this in the future if anything changes. Basically now anywhere that uses timescales has them both initialized and uninitialized. Then the uninitialized ones will automatically be initialized with these new changes as they should have been before.

Reply:
Thank you!